concourse/pipelines/partner-image-validations.jsonnet

```diff
@@ -384,6 +416,13 @@ local ubuntuproposedimages = [
         for family in ubuntudevelimages
       ],
     },
+    {
+      name: 'ubuntu-arm64-devel',
```

Comment:
these groupings create tabs in our ci/cd UI, so it's mostly decoration. i'd rather have them continue to be grouped by prroject, so let's amend this to just have 'ubuntudevelimages + ubuntuarm64develimages' on line 416 above, and delete this block

Reply:
Done.